### ISSUE TITLE
caliper benchmark generator cannot be installed via npm

### ISSUE THREAD

**Issue by mxevgenis** [hyperledger/caliper#832]
I have followed the guide in https://hyperledger.github.io/caliper/vNext/benchmark-generator/ and in the second step where i try to install the generator-caliper i face the issue below:

0 info it worked if it ends with ok
1 verbose cli [ '/usr/bin/node',
1 verbose cli   '/usr/bin/npm',
1 verbose cli   'install',
1 verbose cli   '-g',
1 verbose cli   '@hyperledger/generator-caliper' ]
2 info using npm@6.14.4
3 info using node@v10.20.1
4 verbose npm-session dfbe2f77ade8a42c
5 silly install loadCurrentTree
6 silly install readGlobalPackageData
7 http fetch GET 404 https://registry.npmjs.org/@hyperledger%2fgenerator-caliper 1030ms
8 http fetch GET 404 https://registry.npmjs.org/@hyperledger%2fgenerator-caliper 819ms
9 silly fetchPackageMetaData error for @hyperledger/generator-caliper@latest 404 Not Found - GET https://registry.npmjs.org/@hyperledger%2fgenerator-caliper - Not found
10 timing stage:rollbackFailedOptional Completed in 2ms
11 timing stage:runTopLevelLifecycles Completed in 1872ms
12 verbose stack Error: 404 Not Found - GET https://registry.npmjs.org/@hyperledger%2fgenerator-caliper - Not found
12 verbose stack     at res.buffer.catch.then.body (/usr/lib/node_modules/npm/node_modules/npm-registry-fetch/check-response.js:104:15)
12 verbose stack     at process._tickCallback (internal/process/next_tick.js:68:7)
13 verbose statusCode 404
14 verbose pkgid @hyperledger/generator-caliper@latest
15 verbose cwd /home/user/caliper-benchmarks
16 verbose Linux 4.4.0-116-generic
17 verbose argv "/usr/bin/node" "/usr/bin/npm" "install" "-g" "@hyperledger/generator-caliper"
18 verbose node v10.20.1
19 verbose npm  v6.14.4
20 error code E404
21 error 404 Not Found - GET https://registry.npmjs.org/@hyperledger%2fgenerator-caliper - Not found
22 error 404
23 error 404 '@hyperledger/generator-caliper@latest' is not in the npm registry.
24 error 404 You should bug the author to publish it (or use the name yourself!)
25 error 404 Note that you can also install from a
26 error 404 tarball, folder, http url, or git url.
27 verbose exit [ 1, true ]
From what i saw the packet is missing from the site. Is there a new location of the information??

**Comment by fbalwy** [comment]
I tried to install the generator using`npm install -g @hyperledger/generator-caliper` but got the following error:

```
% npm install -g @hyperledger/generator-caliper

npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@hyperledger%2fgenerator-caliper - Not found
npm ERR! 404 
npm ERR! 404  '@hyperledger/generator-caliper@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/fbalwy/.npm/_logs/2020-05-11T04_43_47_611Z-debug.log
```

**Comment by Lynn14m** [comment]
Seems that the generator hadn't yet been published to the npm registry so that is probably the reason it isn't showing up. The name of the generator in the package.json also needs to be changed from `@hyperledger/generator-caliper` to `generator-caliper`. I'll go ahead and make a PR for that then get the generator published!

**Comment by nklincoln** [comment]
now published at https://www.npmjs.com/package/@hyperledger/generator-caliper

There is currently only an unstable version - not sure if this will be collected by the yoeman agent that scrapes npm

**Comment by aklenik** [comment]
@nklincoln @Lynn14m I guess this issue is resolved then, right?

**Comment by nklincoln** [comment]
agree with above -it is now obtainable on npm, an associated issue with yeoman discovering the published package is being tracked in #920
